File: split-bill-api/api/src/main/java/com/api/dto/user/UserDto.java
package com.api.dto.user;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UserDto {
    Integer id;
    Integer groupId;
    String name;
    String photoUrl;
    Long dateCreated;
    Integer uniqueId;
}

File: app/src/main/java/com/shashankbhat/splitbill/database/local/dto/users/UserDto.kt
package com.shashankbhat.splitbill.database.local.dto.users

import java.io.Serializable

@kotlinx.serialization.Serializable
data class UserDto(
    var name: String,
    var groupId: Int,
    var id: Int? = null,
    var photoUrl: String? = null,
    var dateCreated: Long? = System.currentTimeMillis(),
    var uniqueId: Int? = null
) : Serializable